Data/src/SQLChannel.cpp

```diff
@@ -235,6 +232,7 @@ void SQLChannel::run()
 	{
 		try
 		{
+			sleepTime = 100;
```

Comment:
warning: likely to cause merge conflict with https://github.com/pocoproject/poco/pull/4530